ucm2/sof-soundwire/Hdmi.conf

```diff
@@ -10,6 +10,7 @@ If.hdmi1 {
 			Comment "HDMI1/DP1 Output"
 
 			EnableSequence [
+				cset "name='PGA6.0 6 Master Playback Volume' 32"
```

Comment:
NAK, the volume for HDMI should be full or even removed.

Reply:
@plbossart the volume of 32 is the 0db for PGA volume. I'm planning move such initial setting into SectionVerb in HiFi.conf. What do you think?